pom.xml

```diff
@@ -34,7 +34,7 @@
   </parent>
 
   <artifactId>bouncycastle-api</artifactId>
-  <version>${revision}${changelist}</version>
+  <version>2.30.${revision}-${changelist}</version>
```

Comment:
Why `2.30` and not simply `3`?

Reply:
because then we would need to wait for bouncycastle v4 rather than a v3 (or a v > 2.30).  https://github.com/jenkinsci/bouncycastle-api-plugin/pull/133#discussion_r1413947814 

Granted we are a way of v2 let alone v3 or a v4.   the earth may have imploded by the time we get either...

Reply:
…if you cared about

> we would then be able to remove the prefix

that is.